src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -219,6 +224,145 @@ namespace OpenMS
     return library_;
   }
 
+
+  Size FeatureFinderIdentificationAlgorithm::addOffsetPeptides_(vector<PeptideIdentification>& peptides, double offset)
+  {
+    // WARNING: Superhack! Use unique ID to distinguish seeds from real IDs. Use a mod that will never occur to
+    // make them truly unique and not be converted to an actual modification.
+    const String pseudo_mod_name = String(10000);
+    AASequence some_seq = AASequence::fromString("XXX[" + pseudo_mod_name + "]");
+
+    vector<PeptideIdentification> offset_peptides;
+    offset_peptides.reserve(peptides.size());
+    Size n_added{};
+    for (const auto & p : peptides) // for every peptide (or seed) we add an offset peptide
+    {
+      /*
```

Comment:
Commented out code did lead to slightly worse discrimination (probably decoys are to easy to distinguish now). I left this in for now in case I missed something obvious here.